tests/integration/tester/src/charm.py

```diff
@@ -98,7 +98,7 @@ def _tester_layer(self):
             "PORT": self.config["port"],
             "HOST": self.config["host"],
             "APP_NAME": self.app.name,
-            "TEMPO_ENDPOINT": str(self.tracing.otlp_grpc_endpoint) or "",
+            "TEMPO_ENDPOINT": str(self.tracing.otlp_http_endpoint) or "",
```

Comment:
```suggestion
            "TEMPO_ENDPOINT": str(self.tracing.otlp_http_endpoint or ""),
```
But should it be / is it necessary for the user to convert a `None` to a `""`?

Reply:
besides, `str(None)` will be `"None"` and therefore truthy, so that's a bug either way :)

Reply:
Nice catch! Fixed.